### PR TITLE
Add doc for Chinese LM.

### LINKED ISSUE #365: Complete document for LMs
Due to the problem of paddle cloud, will update the url of large LM later.

### PR DESCRIPTION
**Pull request by pkuyym**
Fix #365

### CODE REVIEW

File: deep_speech_2/README.md [hunk 1]
```diff
@@ -459,10 +459,11 @@ Mandarin | [Internal Mandarin Model](to-be-added) | Baidu Mandarin Dataset | 291
 
 #### Language Model Released
 
-Language Model | Training Data | Token-based | Size | Filter Configuraiton
-:-------------:| :------------:| :-----: | -----: | -----------------:
-[English LM](http://paddlepaddle.bj.bcebos.com/model_zoo/speech/common_crawl_00.prune01111.trie.klm) |  To Be Added | Word-based | 8.3 GB | To Be Added
-[Mandarin LM](http://cloud.dlnel.org/filepub/?uuid=d21861e4-4ed6-45bb-ad8e-ae417a43195e) |  To Be Added | Character-based | 2.8 GB | To Be Added
+Language Model | Training Data | Token-based | Size | Description
+:-------------:| :------------:| :-----: | -----: | :-----------------:
+[English LM](http://paddlepaddle.bj.bcebos.com/model_zoo/speech/common_crawl_00.prune01111.trie.klm) |  [en.00.deduped.xz](http://web-language-models.s3-website-us-east-1.amazonaws.com/ngrams/en/deduped/en.00.deduped.xz) | Word-based | 8.3 GB | pruned with 0 1 1 1 1 <br/> about 1.85 billion n-grams <br/> 'trie'  binary with '-a 22 -q 8 -b 8'
```
Comment (xinghai-sun) [inline deep_speech_2/README.md]: 1. “Large internal data" --> "Internal Data"
2. "pruned ..." --> "Pruned ... ;"
3. "about ... " --> "About ...;"
4. left-alignment for the last column 
5. "en.00.deduped.xz" --> "CommonCrawl(en.00)"

**Review by xinghai-sun — CHANGES_REQUESTED** [review]
Almost LGTM.

File: deep_speech_2/README.md (hunk 1 above)
Comment (xinghai-sun) [reply deep_speech_2/README.md]: Description --> Descriptions

Comment (pkuyym) [reply deep_speech_2/README.md]: Done.

**Review by xinghai-sun — APPROVED** [review]

**Pull request merged**